File: src/main/java/bgu/spl/net/impl/Frames/ClientToServer/ConnectFrame.java
package bgu.spl.net.impl.Frames.ClientToServer;

public class ConnectFrame {

    String msg;
    int version;

    public ConnectFrame(int version) {
        this.version = version;
    }


    void send() {
        msg = "CONNECTED\n version:"+version+"\n\n ^@";
        System.out.println(msg);
    }
}

File: Boost_Echo_Client/src/connectFrame.cpp
//
// Created by eladpr@wincs.cs.bgu.ac.il on 14/01/2020.
//

#include <iostream>
#include "../include/connectFrame.h"


connectFrame::connectFrame(string host,string name,string password){
    this->msg="CONNECT\naccept-version:1.2\nhost:" +host+"\nlogin:"+name+"\npasscode:"+password+"\n\n^@";
}

string connectFrame::toString() {
    return this->msg;
}
